include/odp/api/spec/pool_types.h

```diff
@@ -416,6 +416,30 @@ typedef struct odp_pool_param_t {
 	/** Pool type */
 	odp_pool_type_t type;
 
+	/** Parameters for user area initialization */
+	struct {
+		/** User area initialization function
+		 *
+		 *  Application defined user area initialization function to be
+		 *  called for each event of the pool. Ignored if user area persistence
+		 *  is not supported (odp_pool_capability_t::uarea_persistence) or
+		 *  pool will not have any user area. Use NULL if initialization is
+		 *  not needed. The default value is NULL.
+		 *
+		 *  @param uarea    Pointer to the user area of an event
+		 *  @param size     User area size
+		 *  @param args     Pointer to application defined arguments
+		 *  @param index    Index of the event (0..num events in pool - 1), not
+		 * 		    necessarily in order
+		 */
+		void (*init_fn)(void *uarea, uint32_t size, void *args, uint32_t index);
+
+		/** Pointer to application defined arguments to be passed to every call
+		 *  of init_fn. Use NULL if no data. The default value is NULL.
```

Comment:
"if no data" -> "if no arguments"

Sentence could be also removed, as it's user defined and default value is defined as NULL.

Reply:
Removed in v3.